src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -34,21 +33,18 @@
 {
     use ClassInfoTrait;
 
-    private $propertyNameCollectionFactory;
-    private $propertyMetadataFactory;
     private $itemDataProvider;
     private $routeNameResolver;
     private $router;
-    private $propertyAccessor;
+    private $itemIdentifiersExtractor;
 
     public function __construct(PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, ItemDataProviderInterface $itemDataProvider, RouteNameResolverInterface $routeNameResolver, RouterInterface $router, PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
We should put a reminder somewhere that (when we can break things again) we need to remove those injections and rather directly inject ItemIdentifiersExtractor.